components/brave_wallet/browser/fil_response_parser.cc

```diff
@@ -29,4 +30,34 @@ bool ParseFilGetTransactionCount(const std::string& raw_json, uint64_t* count) {
   return base::StringToUint64(value, count);
 }
 
+bool ParseFilEstimateGas(const std::string& raw_json,
```

Comment:
Please update this based on https://github.com/brave/brave-core/pull/12973/commits/5621c711f0a25203b3b8ed6c48baf7630f54cb4d.

Reply:
done